portfolio/src/main/webapp/index.html

```diff
@@ -2,45 +2,77 @@
+    <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16.png">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>Austin Teshuba</h1>
-
-      <img class="img-primary" src="/images/headshot.jpeg" alt="Profile Picture">
-
-      <p>
-          Hi! Welcome to my portfolio.
-
-      <a href="https://github.com/austinteshuba" target="_blank">Github</a>
-
-      <p> 
-          I'm a rising 3rd-year software engineering student at Western University.
-
-      <p>
-          I’m also a technologist knowledgeable in web development with Angular, 
-          iOS development with Swift, data analysis and ML with Python, algorithms, 
-          object oriented design, and much more.
 
-      <p>
-          Most importantly, I’m a curious person and a lightening fast learner. 
-          Whether I’m mastering web development to complete a pressing project at Deloitte, 
-          or figuring out Twilio and Microsoft Azure in 24 hours at MHacks, 
-          I always approach new problems with a positive attitude and relentless determination.
-
-      <p> 
-          If you're solving big problems with even bigger impacts, please reach out!
-          I'd love to chat more about how I can help.
-
-      <p>
-          What am I doing right now?
-      </p>
-
-      <button onclick="addActivity()">Let's find out</button>
-      <div id="activity-container"></div>
+  <body>
+    <div class="section-shadow-wrapper">
+      <header class="header-primary">
+        <div class="row u-text-shadow">
+          <div class="col-1-of-3 u-padding-sm header-container-left">
+            <div class="u-center u-text-center">
+              <img class="img-circle u-box-shadow" src="/images/headshot.jpeg" alt="Profile Picture">
+              <div class="caption-lg u-text-shadow">
+                Welcome to My Portfolio
+              </div>
+              <div class="icons">
+                <a href="https://www.linkedin.com/in/ateshub" target="_blank" >
+                  <img src="/images/linkedin.png" alt="LinkedIn Logo">
+                </a>
+                <a href="https://www.github.com/austinteshuba" target="_blank" >
+                  <img src="/images/github.png" alt="Github Logo">
+                </a>
+                <a href="https://www.devpost.com/teshuba" target="_blank" >
+                  <img src="/images/link.png" alt="Devpost Link">
+                </a>
+              </div>
+            </div>
+          </div>
+          <div class="col-2-of-3 u-text-right header-container-right">
+            <div class="heading-primary">
+              <h1>Austin Teshuba</h1>
+              <h2>Welcome to my Portfolio!</h2>
+            </div>
+            <div class="heading-secondary">
+              <h3>Do you need someone passionate about:</h3>
+              <h3 id="passions">
+                <span class="blinking-cursor">|</span>
+              </h3>
```

Comment:
Headings (h1, h2, h3, etc.) should be used in a semantically meaningful way. It doesn't really make sense to make the passions a heading, especially when it's dynamically generated by javascript.

Imagine that your site is viewed without any CSS styling or javascript - what do you want the reader to see? The reader may be a search engine or another automated tool. This hypothetical view of your page doesn't have to be pretty, but it does have to make sense.

(This applies to a lot of the other uses of headings on this page as well.)

Ideally, your passions would just be a hidden HTML list embedded in the page, and the JS would take care of showing them one at a time. However, don't worry about making that change now.

Reply:
Agreed. I ended up using p tags with some helper classes. I want them to have the same visual appearance as headers, but semantically it made no sense. I'll keep an eye on this for the future - thanks for the feedback!

Reply:
Also, I'm going to look into the hidden HTML list embedded in the page. Never seen/used that before, but sounds like a great idea. Stay tuned on next PR (of week 2 portfolio changes)

Reply:
@antmar Hey! I looked into the HTML list and it seems possible, but I was wondering why this is a preferred implementation? While holding all the text content in HTML makes more semantic sense, it might just be more trouble than it's worth once readability and complexity is factored in. Hidden elements are also ignored by screen readers so I'm not sure if there's an SEO advantage either.

Please let me know if there's something I'm missing - thanks!